### DISCUSSION TITLE
Card: Libyan civil war

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#9] [Ideas]
If the phrase "Libya is no longer an oil production country" is written out outside of parenthesis it reads more like an effect of the card itself, while in fact it is the canceling of the active effect that reverts Libya from being an Oil Production country back to not being one. Thus this resulting effect should be expressed as a reminder, within parenthesis.

However, another card, 'Hezbollah', cancels an Event effect, 'South Lebanon Conflict'. That card does not specify the resulting reversed effect, which is fortunate since it would require a non-trivial description to express the reverse of "US cannot lose Influence in Lebanon, and no other player can Prevail there".

This is why my suggestions does not include a reminder in the 'Libyan civil war' text, and provides a comment questioning whether it should be present. If it is to be included, then 'Hezbollah' should have a similar, but perhaps simple, reminder.

**Comment by rodrigoSantamaria** [comment]
Either solution is ok to me :)

**Comment by RebelusQuo** [comment]
I will choose the alternative with less text :)
